requests_app/models.py

```diff
@@ -0,0 +1,609 @@
+from django.db import models
+
+
+class Panel(models.Model):
+    """Defines a single internal panel"""
+
+    # this is the PanelApp Panel id itself
+    external_id = models.TextField(
+        verbose_name="external panel id", max_length=255, null=True
+    )
+
+    # metadata
+    panel_name = models.TextField(verbose_name="Panel Name", max_length=255)
+
+    panel_source = models.TextField(
+        verbose_name="panel source",
+        max_length=255,
+    )
+
+    panel_version = models.CharField(
+        verbose_name="panel version",
+        max_length=255,
+        null=True,
+    )
+
+    # reference genome
+    grch37 = models.BooleanField(verbose_name="grch37", default=True)
+    grch38 = models.BooleanField(verbose_name="grch38", default=True)
+
+    # whether panel is created from test directory
+    test_directory = models.BooleanField(
+        verbose_name="created from test directory",
+        null=True,
+        default=False,
+    )
+
+    # whether panel is customized
+    custom = models.BooleanField(
+        verbose_name="custom panel",
+        null=True,
+        default=False,
+    )
+
+    # creation date
+    created_date = models.DateField(
```

Comment:
you could combine them https://docs.djangoproject.com/en/4.2/ref/models/fields/#datetimefield